site-src/concepts/versioning.md

```diff
@@ -14,23 +14,39 @@ new features and resources start in the Experimental release channel. From that
 point, these may graduate to the Standard release channel or be dropped from the
 API entirely.
 
+To get a new item into the Experimental channel, you need:
+* A Who/What/Why [GEP](../geps/overview.md) merged as Provisional
+* Implementations willing to sponsor the item by agreeing to implement it as an 
+experimental feature
+  * Items affecting north-south aspects of Gateway API require three sponsors
+  * Items affecting only east-west aspects require two sponsors
+
+The GEP can be merged into Provisional before soliciting sponsors. Once an item 
+moves from Provisional to Implementable, it will be automatically dropped if it 
+doesn't make any significant progress in six months. This process is handled 
+with a public review, typically at the first meeting of every month.
```

Comment:
I'm worried that this will go stale - any chance we could put something like "There are requirements for each level change, and those are documented at <the GEP overview doc>". That way, if we update the GEP overview doc with the process, we don't need to update it here as well.

Reply:
I think the right answer here is a link to https://gateway-api.sigs.k8s.io/geps/overview/ and, of course, making sure that that document reflects reality.